.github/workflows/build-docs.yml

```diff
@@ -91,3 +91,49 @@ jobs:
           force_orphan: ${{ env.CLEAN_BRANCH }}
           user_name: 'TARDIS Bot'
           user_email: 'tardis.sn.bot@gmail.com'
+
+      - name: Find comment
+        uses: peter-evans/find-comment@v1
+        id: fc
+        with:
+          issue-number: ${{ github.event.number }}
+          body-includes: Hi, human.
+        if: always() && github.event_name == 'pull_request_target'
+
+      - name: Post comment (success)
+        uses: peter-evans/create-or-update-comment@v1
+        with:
+          token:  ${{ secrets.GITHUB_TOKEN }}
+          issue-number: ${{ github.event.number }}
+          comment-id: ${{ steps.fc.outputs.comment-id }}
+          edit-mode: replace
+          body: |
+            *\*beep\* \*bop\**
+
+            Hi, human.
+
+            The **`${{ github.workflow }}`** workflow has **succeeded** :heavy_check_mark:
+            
+            [**Click here**](${{ env.URL }}) to see your results.
+        env:
+          URL: https://${{ github.repository_owner }}.github.io/${{ github.event.repository.name }}/pull/${{ github.event.number }}/index.html
+        if: success() && github.event_name == 'pull_request_target'
+
+      - name: Post comment (failure)
+        uses: peter-evans/create-or-update-comment@v1
+        with:
+          token:  ${{ secrets.GITHUB_TOKEN }}
```

Comment:
tardis bot token, right?
Is this saved as a secret?

Reply:
This line works in other workflows, although I don't know the direct answer to your question off the top of my head.

Reply:
I think we should use the `BOT_TOKEN` here. This also needs to be added in the repo settings.

Reply:
> I think we should use the `BOT_TOKEN` here. This also needs to be added in the repo settings.

Is [this](https://github.com/tardis-sn/stardis/pull/80/commits/2933599237dad1c90dbaf4b7ca3c8949ef90c418) what you had in mind?

Reply:
yes! thank you.